src/components/shared/dots/dots.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+import './dots-extend.scss';
+
+const SurveyTableDot = styled.div`
+      position: relative;
+      background-color: #3e425a;
+      width: 0.5rem;
+      height: 0.5rem;
+      border-radius: 0.5rem;
+      margin: 0.5rem;
+      margin-top: ${({First}) => First && '1.2rem'};
+      background-color: ${({Selected}) => (Selected) && '#eaeaea'};
+`;
+
+
+const Dots = ({onDotsClick, selected, opened}) => {
+    let classAnimationName = '';
+    if (opened === 'open') classAnimationName = 'dots-container-down';
+    if (opened === 'close') classAnimationName = 'dots-container-up';
```

Comment:
It's a little bit weird to have prop `opened="close"` and you have only two states of `opened`, so `boolean` value would be the best in this case.

Reply:
I looked for the usage of Dots component, and `opened` prop will always be either `open` or `close` so I don't quite understand.
```jsx
<Dots opened={survey.opened ? 'open' : 'close'} onDotsClick={this.onDotsClicked} selected={survey.opened}/>
```